app/Model/User.php

```diff
@@ -1324,6 +1324,22 @@ public function registerUser($added_by, $registration, $org_id, $role_id) {
         }
     }
 
+    /**
+     * @param array $user
```

Comment:
Please state what the function does (or should do)

Reply:
Thanks, done.